CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [unreleased]
+### Added
+- producer & consumer creation not allowed when domain is not available
+
```

Comment:
While you are touching this code perhaps you could fix this also? 

Renaming `stream` field in the classes `ProcessInputStream` and `ProcessOutputStream`:
`StreamKey stream;` -> `StreamKey streamKey;`

Renaming `stream` field in the classes `ProcessInputStreamInput` and `ProcessOutputStreamInput`:
`StreamKeyInput stream;` -> `StreamKeyInput streamKey;`

```suggestion
### Changed
- Fixed `streamKey` field name in `ProcessInputStream` and `ProcessOutputStream`
- Fixed `streamKey` field name in `ProcessInputStreamInput` and `ProcessOutputStreamInput`
```

Reply:
done .